### ISSUE TITLE
[ISSUE] Can't destroy resources with mandatory owners

### ISSUE THREAD

**Issue by Doe-Ed** [databrickslabs/terraform-provider-databricks#303]
Hi there,

I created a simple job and assigned ("CAN_MANAGE") permissions to a group. When I try to destroy the job, I get "Error: The job must have exactly one owner. (400 on /api/2.0/preview/permissions/jobs/266)"

### Terraform Version
Terraform v0.12.26
+ provider.aws v2.70.0
+ provider.databricks v0.2.4

### Affected Resource(s)
- databricks_permissions (for clusters and jobs)

### simple_job.tf:
```hcl
resource "databricks_job" "simple_job" {
  name = "simple_job"

  notebook_task {
    notebook_path = "/example/notebook"
  }

  new_cluster {
    num_workers = 1
    spark_version = "6.5.x-scala2.11"
    node_type_id  = "i3.xlarge"
  }
}

resource "databricks_permissions" "simple_job" {
  job_id = databricks_job.simple_job.id

  access_control {
    group_name = "DS"
    permission_level = "CAN_MANAGE"
  }
}
```

### Actual Behavior
Returns error: "Error: The job must have exactly one owner.
(400 on /api/2.0/preview/permissions/jobs/266)"

### Steps to Reproduce
1. `terraform apply`
2. `terraform destroy`

**Comment by nfx** [comment]
@Doe-Ed , please update bug title and description. 

Some comments on config as well:

```hcl
access_control {
    # omit group name parameter completely if you don't use it. Strong recommendation, though, is to assign permissions to groups, not to individual users.
    group_name = null

    # is this equal to "display name" of your user?
    user_name = "nuriyan@yotpo.com"

    permission_level = "CAN_MANAGE"
  }
```

**Comment by Doe-Ed** [comment]
@nfx Thanks for the rapid reply!

1. Updated bug description
2. Changed the example per your comments to follow best practices- permission is now given at the Group level.

Note: I am able to create the job and assign the correct permission to the ("DS") group. It is when I try to destroy it that I get the error.

**Comment by nfx** [comment]
volunteered by @yessawab

**Comment by nfx** [comment]
Fixed in #344 , new documentation is a bit more comprehensive now: https://github.com/databrickslabs/terraform-provider-databricks/blob/master/docs/resources/permissions.md#Job-usage

**Comment by nfx** [comment]
@Doe-Ed released in 0.2.7
